app/api/schema/events.py

```diff
@@ -24,32 +24,6 @@ class Meta:
         self_view_many = 'v1.event_list'
         inflect = dasherize
 
-    @validates_schema(pass_original=True)
-    def validate_date(self, data, original_data):
-        if 'id' in original_data['data']:
```

Comment:
Are you sure it is correct replacement of this statement?

Reply:
The whole idea was to add starts_at and ends_at in data if it is missing while event updation. 
But now I think we should validate date only if it is being updated. Otherwise unwanted errors will arise. 
I think I should call `validate_date` only when event is being created or date is being updated. Does this sound good to u?

Reply:
Yes

Reply:
@iamareebjamal updated the logic. Please take a look